docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+    "signature": "RSA signature-1"
+  }
+]
+```
+
+An ID Token is a type of JWT used in the OpenID Connect protocol by an OP (OpenID Provider) to make claims about an identity. An OP is simply what an IDP (Identity Provider) is called in OpenID Connect. Here is an example ID Token issued by `https://issuer.example.com` making claims about the subject `alice@example.com`.
+
+```JSON
+{"payload":{
+    "iss": "https://issuer.example.com",
+    "aud": "audience-id",
+    "sub": "104852002444754136271",
+    "nonce": "fsTLlOIUqtJHomMB2t6HymoAqJi-wORIFtg3y8c65VY",
+    "email": "alice@example.com"},
+"signatures":[
+  {
+    "protected":{
+      "alg": "RS256",
+      "kid": "1234",
+      "typ": "JWT"
+    },
+    "signature": "GqjU... (Issuer's signature)"
+  }
+]}
+```
+
+## PK Tokens
+
+Now let's look at what happens when we extend this ID Token with additional signatures to create a PK Token. A PK Token will have at least have at least two signatures but can have more than two signatures.
+
+```JSON
+{"payload":{
+    "iss": "https://issuer.example.com",
+    "aud": "audience-id",
+    "sub": "104852002444754136271",
+    "nonce": "fsTLlOIUqtJHomMB2t6HymoAqJi-wORIFtg3y8c65VY",
+    "email": "alice@example.com"},
+"signatures":[
+  {
+    "protected":{
+      "alg": "RS256",
+      "kid": "1234",
+      "typ": "JWT"
+    },
+    "signature": "GqjU... (Issuer's signature)"
+  },
+  {
+    "protected": {
+    "alg": "ES256",
+    "typ": "CIC",
+    "rz": "b9522b5c4cff90687ec6787236184659e077a619b82827227114108440fec26a",
+    "upk": {
+        "alg": "ES256",
+        "crv": "P-256",
+        "kty": "EC",
+        "x": "cvqyUFNs1OUdRcDSmzJfS7ynuTHAjlDqoeinCZy_r1Q",
+        "y": "Whl5jJUIz7ujFvlB5Hzhaz6DIlpyWQmIIA3J7VMj53o"
+    }},
+    "signature":"TBPZa...  (Clients's signature)"
+  },
+  {
+    "protected":{
+      "alg": "ES256",
+      "auth_time": 1722113589,
+      "eid": "6e38311685191ac62e8647e8263f98ad1d57752dab14a5b83298c0d70fe19942",
+      "exp": 1722117189,
+      "iat": 1722113589,
+      "iss": "https://mfa-cosigner.openpubkey.com",
+      "kid": "7890",
+      "nonce": "a958dd0574393cc3fa0e423b4d060009b44ecc710d1ff7af0e4cc74b9fc99649",
+      "ruri": "http://localhost:54435/mfacallback",
+      "typ": "COS"
+    },
+    "signature":"hiyDh... (Cosigner's signature)"
+  }
+]}
+```
+
+### Protected Header Claims
+
+Protected headers in JWS [(RFC-7515 Section 2)](https://datatracker.ietf.org/doc/html/rfc7515#section-2) are used in JWS as a place to put signature header parameters. That is metadata about the signature necessary to verify the signature. For instance parameters such as, `alg` which specifies the algorithm used to generate the signature or `kid` which is the Key ID of the public key used to verify the signature.  In PK Tokens we use protected headers in a new way namely to store claims the signature is making about the identity. Next, we will look at why we need to use protected headers in this new way.
```

Comment:
Suggestion: simplify to "That is metadata necessary to verify the signature."

Reply:
Wording: "For instance, parameters such as, `alg` specifies the algorithm used to generate the signature amd `kid`, which is the Key ID of the public key used to verify the signature, may be included as metadata.